### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by rotsen18**

### CODE REVIEW

**Review by KatiukhaO — APPROVED** [review]

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,64 @@
...
+        )
+
+    def __sub__(self, other):
+        return Vector(
+            x=self.x - other.x,
+            y=self.y - other.y,
+        )
+
+    def __mul__(self, other):
+        if isinstance(other, float) or isinstance(other, int):
```
Comment (TPolina) [inline app/main.py:22]: you can shorten this to `isinstance(other, (int, float))`

**Review by TPolina — APPROVED** [review]
